### DISCUSSION TITLE
Question: How to place hold on a card and capture the payment later

### DISCUSSION THREAD

**Discussion by toklive** [flutter-stripe/flutter_stripe#513] [Q&A]
Hello Team - 

We are planning to use Card Payment, Apple Pay and Google pay options using the authorize and capture later option documented here : https://stripe.com/docs/payments/capture-later

How we can achieve the same using flutter_stripe package ? Any guidelines would be very helpful. Also it is not clear from stripe documentation that whether the Apple Pay and Google Pay are supported for capture later method. Thank you!

**Comment by jonasbark** [comment]
Hi! We recommend contacting Stripe directly using their support system or discord.

**Reply by toklive** [reply]
Thank you, @jonasbark. I have asked about the same in Discord and they don't know about the flutter package. It would be helpful if someone can help us with the below use case and direct how to achieve the below using flutter_stripe package. 

we are building a marketplace app and need to implement the payment feature same as uber/stripe. We are using stripe connected express product. We need to let the customers choose apple pay, google pay, and any credit cards as the payment methods and they should be able to save the payment option with a default payment method as their choice. Whenever they request for service in our platform, we need to use the default payment method and need to authorize them to put a hold on a certain amount and capture the actual amount once the service is delivered. We shouldn't be showing the payment sheet and asking for the authorization all the time (same as Uber and Lyft), customers can choose other payment methods if they want to during this time. We believe we have to use this option: https://stripe.com/docs/payments/capture-later , Our app is being built using flutter and using the flutter_stripe package. No idea how to implement this feature with that in a flutter. In our testing in order to the initial confirmation we have to show the payment sheet the authorization amount for putting the hold (which is not something should need to show the customer - same as what lyft/uber does)

**Reply by jonathanaryQB** [reply]
Hi! [@toklive ] Did you find a solution yet?

**Reply by jackforesightmobile** [reply]
Did you ever manage to find a solution for this issue?

**Reply by jay21b** [reply]
I'm also facing same issue. Did anyone find Solution ??

**Comment by Benjith-Zoondia** [comment]
Any updates?
